src/ui/actions/network.ts

```diff
@@ -126,3 +128,38 @@ export function selectAndFetchRequest(requestId: RequestId): UIThunkAction {
     });
   };
 }
+
+export function seekToRequestFrame(
+  requestId: RequestId,
+  frameId: string,
+  cx: Context
+): UIThunkAction {
+  return async (dispatch, getState, { ThreadFront }) => {
+    const state = getState();
+    const request = state.network.requests.find(request => request.id === requestId);
+    const loadedRegions = getLoadedRegions(state);
+
+    // Don't select a request that's not within a loaded region.
+    if (!request || !loadedRegions || !isPointInRegions(loadedRegions.loaded, request.point)) {
+      return;
+    }
+
+    const requestSummary = getSummaryById(state, requestId);
+    if (!requestSummary) {
+      console.error(`Could not find summary for request "${requestId}"`);
+      return;
+    }
+
+    const timeStampedPoint = requestSummary.point;
+    const pause = ThreadFront.ensurePause(timeStampedPoint.point, timeStampedPoint.time);
+    ThreadFront.timeWarpToPause(pause);
+    const frames = (await pause.getFrames())?.filter(Boolean) || [];
```

Comment:
That `.filter(Boolean)` shouldn't be necessary, any reason why you added this? If `getFrames()` returns falsy frames, that would be a bug that should be fixed.

Reply:
I think TS was typing this as `(Frame | undefined)[]`

Reply:
No it types it as `WiredFrame[] | undefined`